app/assets/javascripts/argo.js

```diff
@@ -76,3 +76,26 @@ function filterList() {
 }
 
 $(document).on('keyup', '#filterInput', function(e) { filterList() });
+
+// Provide warnings when creating a collection.
+function collectionExistsWarning(warningElem, field, value) {
+    const client = new XMLHttpRequest();
```

Comment:
Const is introduced in es6, so you can only use it in an es6 file. Otherwise Uglifier will raise an error when you try to deploy.

Reply:
For my education, what indicates an ES6 file?